src/components/functionselect/functionselect.js

```diff
@@ -14,25 +14,72 @@ angular.module('vlui')
 
         scope.func = {
           selected: undefined,
-          list: [undefined]
+          list: {
+            aboveFold: [],
+            belowFold: [] // could be empty
+          },
+          isTemporal: false, // for making belowFold timeUnits single-column
+          isCount: false // hide "more" & "less" toggle for COUNT
         };
 
-        function getFns(type) {
+        // timeUnits for T
+        var timeUnits = {
+          aboveFold: [
+            undefined,
+            'yearmonthdate', 'year', 
+            'quarter', 'month', 
+            'date','day', 
+            'hours', 'minutes', 
+            'seconds', 'milliseconds'
+          ],
+          belowFold: [
+            'yearquarter',
+            'yearmonth',
+            'yearmonthdatehours',
+            'yearmonthdatehoursminutes',
+            'yearmonthdatehoursminutesseconds',
+            'hoursminutes',
+            'hoursminutesseconds',
+            'minutesseconds', 
+            'secondsmilliseconds'
+          ]
+        };
+        timeUnits.all = timeUnits.aboveFold.concat(timeUnits.belowFold);
+
+        // aggregates for Q
+        var aggregates = {
+          aboveFold: [
+            undefined,
+            'min', 'max',
+            'median', 'mean',
+            'sum' // bin is here
+          ],
+          belowFold: [
+            'missing', 'valid',
+            'distinct', 'modeskew',
+            'q1', 'q3',
+            'stdev', 'stdevp',
+            'variance', 'variancep'
+          ] // there is no 'count' for Q
+        };
+        aggregates.all = aggregates.aboveFold.concat(aggregates.belowFold);
```

Comment:
This is exactly why `$watch('fieldDef')` is triggered a second time without aggregation!